File: Android/shared/src/main/kotlin/shared/dto/enums/OperationType.kt
package shared.dto.enums

import kotlinx.serialization.SerialName

enum class OperationType {
    @SerialName("cashIn")
    CASH_IN, // внесение налички
    UNRECORDED,  // неучтенка
    BONUS,  // премия
    SALARY,  // зарплата
    APPOINTMENT //
}

File: Node/src/dto/enums/OperationType.ts
/// Описывает тип Операции (пока финансы особо не работают)
/// - `APPOINTMENT`: Операция связанная с аппоинтментом
/// - `SALARY`     : Операция связанная с зарплатой
/// - `OTHER`      : "Какая-то" операция
export enum OperationType {
    APPOINTMENT = "appointment",
    SALARY = "salary",
    OTHER = "other",
}
